src/ResourceManager/Dns/Commands.Dns/Records/NewAzureDnsRecordSet.cs

```diff
@@ -33,26 +33,36 @@ public class NewAzureDnsRecordSet : DnsBaseCmdlet
         public string Name { get; set; }
 
         [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The zone in which to create the record set (without a terminating dot).", ParameterSetName = "Fields")]
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The zone in which to create the record set (without a terminating dot).", ParameterSetName = "AliasFields")]
```

Comment:
It looks like you are adding multiple parameter sets, so you will need to set the DefaultParameterSetName Property in your Cmdlet attribute

Reply:
updated.